app/containers/MessageBox/index.tsx

```diff
@@ -132,12 +132,8 @@ interface IMessageBoxState {
 
 const forceJpgExtension = (attachment: ImageOrVideo) => {
 	if (isIOS && attachment.mime === 'image/jpeg' && attachment.filename) {
-		const regex = new RegExp(/.heic$/i);
-		if (attachment.filename.match(regex)) {
-			attachment.filename = attachment.filename.replace(regex, '.jpg');
-		} else {
-			attachment.filename += '.jpg';
-		}
+		const regex = new RegExp(/\.[^/.]+$/); // Check from last '.' of the string
```

Comment:
You're making a change for "all types", but you're requesting tests for jpg only.

Plus: Maybe you could take this opportunity to create a separate file for this method and write tests.